packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -150,7 +182,7 @@ class HookshotContent extends React.Component {
     delete customProps.closePortal;
 
     return (
-      <div {...customProps} className={cx(['content', customProps.className])} ref={refCallback}>
+      <div {...customProps} className={cx(['content', customProps.className])} ref={(element) => { this.contentNode = element; this.props.refCallback(element); }}>
```

Comment:
It looks like refCallback was previously using the spread variable from above but was prefixed with `this.props` after the change